components/root/root.jsx

```diff
@@ -214,6 +214,10 @@ export default class Root extends React.PureComponent {
                 },
                 anonymousId: '00000000000000000000000000',
             });
+
+            rudderAnalytics.ready(() => {
+                Client4.setTelemetryHandler(new RudderTelemetryHandler());
```

Comment:
Do you also have to delete the invocation on 186?

Reply:
Good catch. That's what I get for doing a `git add -p` to skip some debugging code